java/flight/src/test/java/org/apache/arrow/flight/TestTls.java

```diff
@@ -29,11 +29,13 @@
 import org.apache.arrow.memory.RootAllocator;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Tests for TLS in Flight.
  */
+@Ignore
```

Comment:
why is this added?  You might need to checkout the testing submodule if this test was failing for you?

Reply:
This is my fault. Sorry about my carelessness.